src/main/java/io/kestra/plugin/weaviate/RawQueryInterface.java

```diff
@@ -0,0 +1,22 @@
+package io.kestra.plugin.weaviate;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+
+public interface RawQueryInterface {
```

Comment:
Same as above, no use to do so here, please remove the interface and move up the doc to the top level class